src/main/java/pl/javastart/task/components/Processor.java

```diff
@@ -0,0 +1,31 @@
+package pl.javastart.task.components;
+
+import pl.javastart.task.exception.IllegalClockSpeedException;
+import pl.javastart.task.printer.Printer;
+
+import java.util.InputMismatchException;
+
+public class Processor extends OverclockableComponent {
+    private static final int TEMPERATURE_INCREASE = 10;
+
+    public Processor(String model, String manufacturer, String serialNo,
+                     int clockSpeed, int temperature, int maxTemperature, Printer printer) {
+        super(model, manufacturer, serialNo, TEMPERATURE_INCREASE, clockSpeed, temperature, maxTemperature, printer);
+    }
+
+    @Override
+    public void overclock() throws InputMismatchException, IllegalClockSpeedException {
+        super.overclock();
+    }
```

Comment:
Jeżeli tutaj jedynie wywołujesz metodę z klasy nadrzędnej, to nie ma potrzeby zapisywania takiej metody